web/app.js

```diff
@@ -182,6 +182,31 @@ class DefaultExternalServices {
   }
 
   static get scripting() {
+    if (PDFJSDev.test("GENERIC")) {
+      const promise = new Promise((resolve, reject) => {
+        const sandboxScript = document.createElement("script");
+        sandboxScript.src = "../build/pdf.sandbox.js";
+        sandboxScript.type = "text/javascript";
+        sandboxScript.async = true;
+        sandboxScript.onload = () => {
+          resolve();
+        };
+        document.head.appendChild(sandboxScript);
+      }).then(() => {
```

Comment:
Please use [`loadScript`](https://github.com/mozilla/pdf.js/blob/55f55f58594b9a6947fecaabf8ef4e3b02002023/src/display/display_utils.js#L531-L546) rather than essentially re-implementing that manually.